scripts/backup.sh

```
@@ -0,0 +1,61 @@
+# Configurar cliente MinIO
+mc alias set local http://localhost:9000 ${MINIO_ROOT_USER:-minioadmin} ${MINIO_ROOT_PASSWORD:-minioadmin123}
+
+# Fazer backup de cada bucket
+for bucket in $(mc ls local --json | jq -r '.key' | sed 's|/||g'); do
+    echo "📁 Fazendo backup do bucket: $bucket"
+    mc mirror local/$bucket $BACKUP_DIR/$MINIO_BACKUP_DIR/$bucket
+done
+
+# Comprimir backups
+echo "🗜️  Comprimindo backups..."
+cd $BACKUP_DIR
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)